.github/workflows/build-apps.yml

```diff
@@ -48,16 +47,17 @@ jobs:
       - name: Run build:wasm
         run: "yarn build:wasm"
 
-      - name: Set nightly version, product name, and icons
-        if: ${{ env.NIGHTLY == 'true' }}
+      - name: Set nightly version, product name, release notes, and icons
+        if: ${{ env.IS_NIGHTLY == 'true' }}
         run: yarn files:flip-to-nightly
 
-      - name: Generate release notes
+      - name: Set release version
+        if: ${{ env.IS_RELEASE == 'true' }}
         run: |
-          echo "$NOTES" > release-notes.md
-          cat release-notes.md
+          export VERSION=${GITHUB_REF_NAME#v}
```

Comment:
[question] what does `#v` signify or do?

Reply:
@franknoirot it chops off the leading 'v'